bundles/io/org.openhab.io.myopenhab/src/main/java/org/openhab/io/myopenhab/internal/MyOpenHAB.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.io.myopenhab.internal;
+
+import org.openhab.core.scriptengine.action.ActionDoc;
+import org.openhab.io.myopenhab.MyOpenHABService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** 
+ * This class provides static methods that can be used in automation rules
+ * for using my.openHAB functionality. 
+ * 
+ * @author Victor Belov
+ * @since 1.3.0
+ *
+ */
+
+public class MyOpenHAB {
+	private static final Logger logger = LoggerFactory.getLogger(MyOpenHAB.class);
+	static MyOpenHABService mMyOpenHABService = null;
+	
+	/**
+	 * Sends a simple push notification to mobile devices of user
+	 * 
+	 * @param userId the my.openHAB user id of the recipient
+	 * @param message the body of the notification
+	 * 
+	 */
+	@ActionDoc(text="Sends a push notification to mobile devices of user with userId")
+	static public void sendNotification(String userId, String message) {
+		logger.debug("sendNotification()");
```

Comment:
should we enhance the statement to log the given message (and userid)? Did you omit these details on purpose?